### PR TITLE
Merge feature branch to develop - Feat/use-fdk-search-service

### PR DESCRIPTION
**Pull request by hegeaal**

### CODE REVIEW

File: deploy/base/deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,33 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: fdk-portal-new-search
```
Comment (NilsOveTen) [inline deploy/base/deployment.yaml:4]:
```suggestion
  name: fdk-portal
```

File: deploy/base/deployment.yaml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  labels:
+    app: fdk-portal-new-search
```
Comment (NilsOveTen) [inline deploy/base/deployment.yaml:6]:
```suggestion
    app: fdk-portal
```

File: deploy/base/deployment.yaml [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: fdk-portal-new-search
```
Comment (NilsOveTen) [inline deploy/base/deployment.yaml:11]:
```suggestion
      app: fdk-portal
```

File: deploy/base/deployment.yaml [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+  strategy:
+    type: RollingUpdate
+  template:
+    metadata:
+      labels:
+        app: fdk-portal-new-search
```
Comment (NilsOveTen) [inline deploy/base/deployment.yaml:17]:
```suggestion
        app: fdk-portal
```

File: deploy/base/deployment.yaml [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+      annotations:
+        prometheus.io/scrape: "false"
+    spec:
+      containers:
+        - name: fdk-portal-new-search
```
Comment (NilsOveTen) [inline deploy/base/deployment.yaml:22]:
```suggestion
        - name: fdk-portal
```

File: deploy/base/deployment.yaml [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+          image: fdk-portal-new-search
```
Comment (NilsOveTen) [inline deploy/base/deployment.yaml:23]:
```suggestion
          image: fdk-portal
```

File: deploy/base/kustomization.yaml [hunk 7]
```diff
@@ -0,0 +1,10 @@
+---
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+  - deployment.yaml
+  - service.yaml
+images:
+  - name: fdk-portal-new-search
```
Comment (NilsOveTen) [inline deploy/base/kustomization.yaml]:
```suggestion
  - name: fdk-portal
```

File: deploy/staging/env.yaml [hunk 8]
```diff
@@ -0,0 +1,131 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: fdk-portal-new-search
```
Comment (valosnah) [inline deploy/staging/env.yaml:5]: fdk-portal

File: deploy/base/kustomization.yaml [hunk 9]
```diff
@@ -0,0 +1,10 @@
...
+    newName: eu.gcr.io/digdir-fdk-infra/fdk-portal-new-search
```
Comment (NilsOveTen) [inline deploy/base/kustomization.yaml]:
```suggestion
    newName: eu.gcr.io/digdir-fdk-infra/fdk-portal
```

File: deploy/base/service.yaml [hunk 10]
```diff
@@ -0,0 +1,16 @@
+---
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    fdk.service: fdk-portal-new-search
```
Comment (NilsOveTen) [inline deploy/base/service.yaml:6]:
```suggestion
    fdk.service: fdk-portal
```

File: deploy/base/service.yaml [hunk 11]
```diff
@@ -0,0 +1,16 @@
...
+  name: fdk-portal-new-search
```
Comment (NilsOveTen) [inline deploy/base/service.yaml:7]:
```suggestion
  name: fdk-portal
```

File: deploy/base/service.yaml [hunk 12]
```diff
@@ -0,0 +1,16 @@
...
+spec:
+  selector:
+    app: fdk-portal-new-search
```
Comment (NilsOveTen) [inline deploy/base/service.yaml:10]:
```suggestion
    app: fdk-portal
```

File: deploy/staging/env.yaml (hunk 8 above)
Comment (NilsOveTen) [inline deploy/staging/env.yaml:5]:
```suggestion
  name: fdk-portal
```

File: deploy/staging/env.yaml [hunk 13]
```diff
@@ -0,0 +1,131 @@
...
+  labels:
+    app: fdk-portal-new-search
```
Comment (NilsOveTen) [inline deploy/staging/env.yaml:7]:
```suggestion
    app: fdk-portal
```

File: deploy/staging/env.yaml [hunk 14]
```diff
@@ -0,0 +1,131 @@
...
+spec:
+  template:
+    spec:
+      containers:
+        - name: fdk-portal-new-search
```
Comment (NilsOveTen) [inline deploy/staging/env.yaml:12]:
```suggestion
        - name: fdk-portal
```

File: deploy/staging/env.yaml [hunk 15]
```diff
@@ -0,0 +1,131 @@
...
+          env:
+            - name: CONTAINER_IMAGE
+              value: eu.gcr.io/digdir-fdk-infra/fdk-portal-new-search:$(GIT_COMMIT_SHA)
```
Comment (NilsOveTen) [inline deploy/staging/env.yaml:15]:
```suggestion
              value: eu.gcr.io/digdir-fdk-infra/fdk-portal:$(GIT_COMMIT_SHA)
```

**Review by NilsOveTen — APPROVED** [review]

**Review by valosnah — APPROVED** [review]

**Pull request merged**